data/boosters/fin-collector.yaml

```diff
@@ -0,0 +1,202 @@
+# Data from: https://magic.wizards.com/en/news/feature/collecting-final-fantasy
+# And https://magic.wizards.com/en/products/final-fantasy/card-image-gallery
+
+pack:
+  foil_common: 3
+  foil_uncommon: 1
+  boosterfun_common_uncommon: 1
+  foil_boosterfun_common_uncommon: 1
+  foil_basic: 1
+  foil_rare_mythic: 1
+  boosterfun_rare_mythic_slot:
+    - non_foil_boosterfun_rare_mythic: 3
+      chance: 753571 # 75.3571%
+    - non_foil_boosterfun_rare_mythic: 2
+      fic_foil_boosterfun_rare_mythic: 1
+      chance: 246429 # 24.6429%
+  through_the_ages:
+    - non_foil_through_the_ages: 1
+      chance: 1 # 50%
+    - foil_through_the_ages: 1
+      chance: 1 # 50%
+  foil_boosterfun_rare_mythic: 1
+
+queries:
+  uncommon_without_cid: "r:u -cid"
+  regular_cid: "e:{set} cid -frame:extendedart"
+  adventure_lands: "e:{set} number:310-314"
+  artist: "e:{set} number:315-323,577"
+  woodblock: "e:{set} number:324-373"
+  character: "e:{set} number:374-405"
+  extended_art_legends: "e:{set} number:421-518"
+  surge_foil_character: "e:{set} number:519-550"
+  boosterfun: "e:{set} number:310-420,577" # Doesn't include extended-art legends
+  borderless_traveling_chocobo: "e:{set} number:406"
+  colored_chocobo: "e:{set} number:551-551d"
+  fic_base_commanders: "e:FIC number:1-8"
+  fic_extended_art: "e:FIC number:101-193"
+  fic_borderless_sagas: "e:FIC number:194-200"
+  fic_borderless_character: "e:FIC number:201-208"
+  fic_surge_foil_borderless_character: "e:FIC number:209-216"
+  fic_secret_rendezvous: "e:FIC number:217-219"
+  fic_extended_art_rares_non_legendary_creatures: "e:FIC number:101-128"
+ 
+sheets:
+  foil_common:
+    foil: true
+    query: "r:c -t:basic"
+    count: 90
+
+  foil_uncommon:
+    foil: true
+    any:
+    - query: "{uncommon_without_cid}"
+      chance: 107676 # 99.7% (997/1000) * 108/109 = 107676/109000
+      count: 108
+    - rawquery: "{regular_cid}"
+      chance: 997 # 99.7% * 0.9% = 997/109000
+      count: 15
```

Comment:
You can do this way simpler using `rate` instead of `chance`. Because each uncommon appears at the same overall rate. The card "Cid" appears just as much as any other uncommon. So you can do this:

```yaml
foil_uncommon:
  foil: true
  any:
  - query: "{uncommon_without_cid}"
    rate: 15
  - rawquery: "{regular_cid}"
    rate: 1
```

This says that each "Cid" counts as 1/15th of the other cards.